source/_docs/z-wave/device-specific.markdown

```diff
@@ -144,19 +144,45 @@ Some models of the Zooz Toggle switches ship with an instruction manual with inc
 
 To provide Central Scene support you need to shut Home Assistant down and modify your `zwcfg_*.xml` file according to the following guides.
 
-### {% linkable_title HomeSeer Switches %}
```

Comment:
Why is this removing/replacing the HomeSeer switches sections?

Reply:
I don't know why the diff looks like that. But if you look at the bottom of what I supposedly added, you'll see the Homeseer stuff is still there.

Reply:
OMG... I'm so sorry, I should not have missed that...